main_GANs.py

```diff
@@ -32,27 +45,38 @@
 
 '''
 
-#model = MLP_GANs_model(mnist_data(rand_rotation = False, max_degree = 90))
-#model.train(num_epochs = 200, lr_x = torch.tensor([0.01]), lr_y = torch.tensor([0.01]),
-#            optimizer_name = 'CGD', verbose = True, label_smoothing = False, single_number = 4) # save_path = ''
-
-#model = DCGANs_model(mnist_data_dcgans(rand_rotation = False, max_degree = 90))
-#model.train(num_epochs = 1, lr_x = torch.tensor([0.01]), lr_y = torch.tensor([0.01]),
-#           optimizer_name = 'Jacobi', verbose = True, label_smoothing = False, single_number = 9) #save_path = ''
-
-#model.save_models()
-
-plt.figure()
-plt.plot([x for x in range(0, len(model.D_error_real_history))],
-         model.D_error_real_history)
-plt.plot([x for x in range(0, len(model.D_error_fake_history))],
-         model.D_error_fake_history)
-plt.plot([x for x in range(0, len(model.G_error_history))],
-         model.G_error_history)
-plt.xlabel('Iterations')
-plt.ylabel('Loss function value')
-plt.legend([
-    'Discriminator: Loss on Real Data', 'Discriminator: Loss on Fake Data',
-    'Generator: Loss'
-])
-plt.savefig('cost_report.png')
+if __name__ == '__main__':
+    arguments = docopt(__doc__, version='Competitive Gradient Descent 0.0')
```

Comment:
Should `0.0` be put in a version somewhere?

Reply:
Yeah, right now it is arbitrary anyway. I guess we could set a separate variable for the version. I think it could be changed later, when the version will have some meaning.